package/Codegen/TSTypes.tt

```diff
@@ -20,7 +28,7 @@ import type {
 <# } #>
 } from './Props';
 
-<# foreach (var ns in Types.Where(t => Util.HasCtor(t) && Util.GetTSNamespace(t) != "").Select(t => Util.GetTSNamespace(t)).Distinct()) { #>
+<# foreach (var ns in Types.Where(t => Util.HasCtor(t) && Util.GetTSNamespace(t) != "").Union(Util.eventArgsTypes.Where(t => Util.GetTSNamespace(t) != "")).Select(t => Util.GetTSNamespace(t)).Distinct()) { #>
```

Comment:
This was necessary because WinUI 2.8 brings in WebView2, and WebView2 has events that use event arg types defined in WebView2's winmd, which don't have public constructors. So even if we add WebView2's winmd to the codegen (and esp. since we don't) then the codegen doesn't export the WebView2 namespace here, which causes the TS build to fail.